ui/src/app/tasks-jobs/tasks/clone/clone.component.html

```diff
@@ -10,27 +10,27 @@ <h3 *ngIf="tasks?.length > 1" class="modal-title">Clone Tasks</h3>
       clrLabelSize="4"
       style="padding: 0; margin: 0"
     >
-      <div style="margin-bottom: 0">This action will create the following <strong>task(s)</strong>:</div>
+      <div style="margin-bottom: 0" [innerHTML]="'tasks.clone.content' | translate"></div>
       <div class="alert alert-danger" *ngIf="form?.hasError('uniqueTaskNames')">
         <div class="alert-items">
           <div class="alert-item static">
             <div class="alert-icon-wrapper">
               <clr-icon class="alert-icon" shape="exclamation-circle"></clr-icon>
             </div>
-            <div class="alert-text">Duplicate task name, please check input names.</div>
+            <div class="alert-text">{{ 'tasks.clone.duplication' | translate }}</div>
           </div>
         </div>
       </div>
       <div *ngFor="let task of tasks; let i = index" class="card" style="padding: 10px 0">
         <div class="card-block">
           <div class="clr-form-control clr-row" style="margin-top: 0; height: 42px">
-            <label class="clr-control-label clr-col-12 clr-col-md-2">Definition:</label>
+            <label class="clr-control-label clr-col-12 clr-col-md-2">{{ 'tasks.main.definition' | translate }}:</label>
```

Comment:
Texts like `Name` and `Definition` are visibile on other pages.
Can we move these to `commons` translations ?